roles/hammerdb/tasks/main.yaml

```diff
@@ -21,8 +21,7 @@
         name: '{{ meta.name }}-workload-{{ trunc_uuid }}'
         namespace: '{{ operator_namespace }}'
       data:
-        workload.tcl: "{{ lookup('template', 'workload.tcl.j2') }}"
-
+ "{{ lookup('template', 'tpcc-workload.tcl.j2') }}"
```

Comment:
indentation error perhaps?

Reply:
yeah, that's what it looks like, let me fix it

Reply:
actually no, it looks fine